### DISCUSSION TITLE
Airtable import [Oct 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6706] [Discord Threads]
> 🙋‍♂️ **soggydoughnuts**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@202054495176359936> Would recommend to try using bug fix release version v0.111.2 (https://github.com/nocodb/nocodb/releases/tag/0.111.2)  
#  
> 👨‍🦲 **dstala**  
#### if you still face issue, let us know by providing details either here or in github issue  
#  
> 🙋‍♂️ **camzanno**  
#### hey <@857998597474484295> I work with Soggy. We're using the docker compose setup for noco, is there a way to force this new version. We're running latest atm  
#  
> 👨‍🦲 **dstala**  
#### https://docs.nocodb.com/getting-started/upgrading  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1149333320656752672)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
